### PR TITLE
Fix hex mode sending

### LINKED ISSUE #21: Hexmode does not work properly
Sending in Hex-Mode garbles all hex values that contain letters.

E.g. sending `FF0102` will result in `FD0102`, `ABCDEF` will result in `FDFDFD`.

### PR DESCRIPTION
**Pull request by CShark**
Fix hexmode sending bugs

Fixes #21

### CODE REVIEW

File: actions.js [hunk 1]
```diff
@@ -89,10 +89,10 @@ export function getActionDefinitions(self) {
 					 * which then escapes character values over 0x7F
 					 * and destroys the 'binary' content
 					 */
-					const sendBuf = Buffer.from(cmd + action.options.id_end, 'latin1')
+					const sendBuf = Buffer.concat([cmd, Buffer.from(action.options.id_end, 'latin1')])
 
 					if (self.config.prot == 'tcp') {
-						self.log('debug', 'sending to ' + self.config.host + ': ' + sendBuf.toString())
+						self.log('debug', 'sending to ' + self.config.host + ': ' + sendBuf.toString('hex'))
```
Comment (istnv) [inline actions.js:95]: Add this 'hex' option to the next 'udp'  log entry.

Comment (istnv) [reply actions.js:95]: Also include an update the `package.json` version number.

Comment (CShark) [reply actions.js:95]: Done

**Comment by istnv** [comment]
@CShark 
Thanks for the patches

**Pull request merged**